ee/common-annotations-1.3.adoc

```diff
@@ -0,0 +1,49 @@
+= Common Annotations 1.3  
+:author:            Tomaz Cerar
+:email:             tcerar@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+
+Commons Annotations 1.3 is a small change update to 1.2 MR.
+Main change is allowing @Repeatble on @DatasourceDefintion and @Priority annotation is allowed to also on methods now.
+We are using our fork of upstream's https://github.com/javaee/javax.annotation
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-900[EAP7-900]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/WFLY-6463[WFLY-6463]
+* https://issues.jboss.org/browse/WFLY-9748[WFLY-9748]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
```

Comment:
Add the contacts. Not sure if this is strictly required for umbrella issues like this, but feel free to add me as QE contact.

Reply:
added